src/main/java/se/kth/dd2480/group25/assignment1/helpers/Constants.java

```diff
@@ -3,6 +3,8 @@
 public final class Constants {
     // Denotes the tolerance when comparing two floating point numbers to account for floating point precision
     public static final double FLOAT_TOLERANCE = 0.0000001d;
+    // Custom definition of pi
+    public static final double PI = 3.1415926535;
```

Comment:
Is there a reason for not using the in-built Math.PI instead?

Reply:
The skeleton code in the assignment defined pi this way, but I'm unsure if it needs to be the exact same or not. Math.PI has a few more decimals, I think.

Reply:
Okay nice, makes sense to follow the spec